py/surveyqa/summary.py

```diff
@@ -189,8 +185,8 @@ def get_summarytable(exposures):
     med_sky = get_median('SKY', exposures)
 
     source = ColumnDataSource(data=dict(
-        nights = list(OrderedDict(Counter(exposures['NIGHT'])).keys()),
-        totals = list(OrderedDict(Counter(exposures['NIGHT'])).values()),
+        nights = list(nights),
+        totals = totals,
```

Comment:
`nights` was created sorted above, and `totals` calculated as we went, thus avoiding another set of calls to `Counter`.  See previous note about `OrderedDict` preserving the order, not ordering (sorting).